06-11-2018/Italent2BoardCreation/src/main/java/com/italent2/service/BoardCreationAndMessagePosting.java

```diff
@@ -0,0 +1,141 @@
+package com.italent2.service;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpResponseException;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.italent2.model.Messages;
+
+public class BoardCreationAndMessagePosting 
+{
+	ResourceBundle rb=ResourceBundle.getBundle("Application");
+	public String sesiionKeyGeneration() throws HttpResponseException, IOException
+	{
+			CloseableHttpClient client = HttpClients.createDefault();
+			HttpPost postRequest=new HttpPost(rb.getString("posturl"));
+			postRequest.addHeader("Authorization","Basic ZGVtbzAxOlN1UHIxKm1aTGk=");
+			HttpResponse response=client.execute(postRequest);
+			int statusCode=response.getStatusLine().getStatusCode();
+			if(statusCode!=200)
+			{
+				throw new RuntimeException("failed with response code"+statusCode);
+			}
+			System.out.println(response);
```

Comment:
dont use System.out.println. start with logs, already sai have idea on this sit with him and setup log4j or log4j2. remove all System.out.println and place logger.info